src/components/Authentication/Register.jsx

```diff
@@ -0,0 +1,199 @@
+            <FormLabel htmlFor="firstName">First Name</FormLabel>
+            <Input
+              id="firstName"
+              bg="white"
+              onChange={({ target }) => setFirstName(target.value)}
+              size="md"
+              value={firstName}
+            />
+          </FormControl>
+          <FormControl>
+            <FormLabel htmlFor="lastName">Last Name</FormLabel>
+            <Input
+              id="lastName"
+              bg="white"
+              onChange={({ target }) => setLastName(target.value)}
+              size="md"
+              value={lastName}
+            />
+          </FormControl>
+          <FormControl>
+            <FormLabel htmlFor="email">OC Habitats Email</FormLabel>
+            <Input
+              id="email"
+              bg="white"
+              onChange={({ target }) => setEmail(target.value)}
+              size="md"
+              value={inviteEmail}
+            />
+          </FormControl>
+          <FormControl>
+            <FormLabel htmlFor="role">Account Type</FormLabel>
+            <Select
+              id="role"
+              onChange={e => setRole(e.target.value)}
+              value={inviteRole}
+              bg="white"
+              disabled
+            >
+              <option value="admin">Admin</option>
+              <option value="super-admin">Super Admin</option>
+              <option value="volunteer">Volunteer</option>
+            </Select>
+          </FormControl>
+          <FormControl>
+            <FormLabel htmlFor="password">Password</FormLabel>
+            <InputGroup>
+              <Input
+                type={showPassword ? 'text' : 'password'}
+                onChange={({ target }) => setPassword(target.value)}
+                bg="white"
+              />
+              <InputRightAddon
+                children={`${showPassword ? 'Hide' : 'Show'}`}
+                onClick={() => setShowPassword(!showPassword)}
+              />
+            </InputGroup>
+          </FormControl>
+          <FormControl>
+            <FormLabel htmlFor="password">Re-enter Password</FormLabel>
+            <InputGroup>
+              <Input
+                type={showCheckPassword ? 'text' : 'password'}
+                onChange={({ target }) => setCheckPassword(target.value)}
+                bg="white"
+              />
+              <InputRightAddon
+                children={`${showCheckPassword ? 'Hide' : 'Show'}`}
+                onClick={() => setShowCheckPassword(!showCheckPassword)}
+              />
+            </InputGroup>
+          </FormControl>
+          {errorMessage && (
+            <Center width="100%">
+              <Text color="red">{errorMessage}</Text>
+            </Center>
+          )}
+          <Text>
+            {/* TODO: REPLACE THESE LINKS WITH ACTUAL URLS */}
+            By continuing, you agree to OC Habitats&apos;s{' '}
+            <a href="https://www.nba.com">Terms & Conditions</a> and
```

Comment:
could you make this oc habitats home page or something at least just in case we forget about this LOL